recipes/trisbm/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "nsbm" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/fvalle1/trisbm/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 99981f5a96c579999011301c163dff92c86209ed4929bc8ba721c919099002db
+
+build:
+  skip: true  # [win]
+  number: 0
+  #script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy
+    - boost-cpp
+    - boost
+    - expat
+    - scipy >=1.0
+    - gmp
+    - cgal-cpp 5.0*
+    - sparsehash 2.0*
+    - cairomm
+    - pycairo
+    - glib
+    - graph-tool
+    - matplotlib
+    - pandas
+    - cloudpickle
```

Comment:
You probably don't need most of these as I guess the Python packages like `matplotlib` are runtime dependencies only.

Reply:
You are right they are not actually needed